src/test/test_snap_mapper.cc

```diff
@@ -573,6 +587,43 @@ class MapperVerifier {
       }
       hoids.clear();
     }
+
+    return ret;
+  }
+  // must be called with lock held
+  // will keep trimming until reaching max_count or failing a call to __trim_snap()
+  void trim_snap_force(snapid_t snapid, int max_count, vector<hobject_t> & out) {
+    int guard = 1000;
+    vector<hobject_t> tmp;
+    unsigned prev_size = 0;
+    while (tmp.size() < (unsigned)max_count) {
```

Comment:
`max_count`'s type can be changed to `size_t` to avoid unsafe conversions.

Reply:
changed to unsigned